qa-tester/assets/src/js/admin-bar/components/container/index.js

```diff
@@ -0,0 +1,241 @@
+				return {
+					label: `${ branch } release branch`,
+					value: branch,
+					origin: 'branch',
+				};
+			} );
+
+			this.setState( {
+				buildOptions: this.state.buildOptions.concat( branchOptions ),
+			} );
+		} );
+	}
+
+	/**
+	 * Append to the default build options a list of PRs that have build zips available for download.
+	 */
+	addPullRequestOptions() {
+		// Retrieve the PRs from GitHub and append them to the list of builds above.
+		getPullRequestsWithBuilds().then( ( results ) => {
+			const prOptions = results.map( ( pr ) => {
+				return {
+					label: `PR #${ pr.number }: ${ pr.title }`,
+					value: pr.number,
+					origin: 'pr',
+				};
+			} );
+
+			this.setState( {
+				isLoading: false,
+				buildOptions: this.state.buildOptions.concat( prOptions ),
+			} );
+		} );
+	}
+
+	/**
+	 * Set the selected build option.
+	 *
+	 * @param {Object} newOption Selected build option.
+	 */
+	handleChangeBuildOption( newOption ) {
+		this.setState( { buildOption: newOption } );
+	}
+
+	/**
+	 * Set whether a dev build is needed or not.
+	 *
+	 * @param {Event<HTMLInputElement>} event Change event.
+	 */
+	handleChangeDevBuild( event ) {
+		const checked = event.target.checked;
+		this.setState( { isDevBuild: checked } );
+	}
+
+	/**
+	 * Switches the AMP plugin to the selected build option.
+	 */
+	handleActivation() {
+		const { isDevBuild, buildOption } = this.state;
+		this.setState( { isSwitching: true } );
+
+		apiFetch( {
+			path: '/amp-qa-tester/v1/switch',
+			method: 'POST',
+			data: {
+				id: buildOption.value,
+				origin: buildOption.origin,
+				isDev: isDevBuild,
+			},
+		} )
+			.then( () => {
+				// @todo Could this be handled better?
+				// Set a 1 second delay before reloading the page to allow for the new plugin to activate without any error.
+				this.pageReloadTimeout = setTimeout( () => {
+					window.location.reload();
+				}, 1000 );
+			} )
+			.catch( ( error ) => {
+				if ( error.message ) {
+					this.showError( error.message );
+				} else {
+					this.showError();
+				}
+			} );
+	}
+
+	/**
+	 * Display an error message.
+	 *
+	 * @param {string} message Message.
+	 */
+	showError( message = '' ) {
+		if ( ! message ) {
+			message = __(
+				'An unknown error occurred while switching build',
+				'amp-qa-tester'
+			);
+		}
+
+		this.setState( {
+			isSwitching: false,
+			error: message,
+		} );
+
+		this.errorTimeout = setTimeout( () => {
+			this.setState( { error: null } );
+		}, 5000 );
+	}
+
+	render() {
+		const {
+			isLoading,
+			isSwitching,
+			error,
+			buildOption,
+			buildOptions,
+		} = this.state;
+
+		if ( isLoading ) {
+			return <p>{ __( 'Loading…', 'amp-qa-tester' ) }</p>;
+		}
+
+		return (
+			<>
+				<BuildSelector
+					buildOption={ buildOption }
+					buildOptions={ buildOptions }
+					onOptionSelect={ this.handleChangeBuildOption.bind( this ) }
+				/>
+
+				{ 'release' !== buildOption.value && (
+					<div className={ 'amp-qa-tester-checkbox' }>
```

Comment:
Since these class names aren't dynamic, they can just be strings.
```suggestion
					<div className="amp-qa-tester-checkbox">
```